### ISSUE TITLE
NVDA reads native emoji shortcode twice

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/emoji-picker-element#8]
For native (not custom) emoji, NVDA+Firefox reads the shortcode name twice when pressing up/down through search results. VoiceOver+Safari does not have this issue.

Not sure why; maybe it's the `title`? I don't plan on getting rid of the `title`, though.

**Comment by t3chguy** [comment]
It looks like it is because of your `aria-label` duplicating the `title`

It looks to read `aria-label` then `title` without a break and omit the text leaf (native emoji)

When I test by getting rid of the `aria-label` it reads the text leaf (NVDA's own description of that emoji) followed by the `title`

I'd argue the best description for screen reader users is the one that their AT gives them as this will be consistent between all the applications they use.

**Comment by nolanlawson** [comment]
Yep, it seems to be because of the `title`. VoiceOver doesn't have the same issue, so they seem to have another implementation.

I can't think of a good solution that doesn't involve getting rid of the title. We can get rid of the title if we implement #1 though.
